pkg/kn/traffic/compute.go

```diff
@@ -276,8 +278,17 @@ func Compute(cmd *cobra.Command, targets []servingv1.TrafficTarget, trafficFlags
 	traffic := newServiceTraffic(targets)
 
 	// First precedence: Untag revisions
+	var errStrings []string
 	for _, tag := range trafficFlags.UntagRevisions {
-		traffic.untagRevision(tag)
+		err = traffic.untagRevision(tag)
+		if err != nil {
+			errStrings = append(errStrings, err.Error())
+		}
+	}
+
+	// Return all errors from untagging revisions
+	if len(errStrings) > 0 {
+		return nil, fmt.Errorf(strings.Join(errStrings, "\n"))
```

Comment:
Please don't add any formatting like `\n` to the end of an error message. Use a `", "` instead as separator.

Reply:
Believe this is addressed by comment below:

https://github.com/knative/client/pull/880#discussion_r437513614